eZ/Bundle/EzPublishRestBundle/Resources/config/routing.yml

```diff
@@ -95,6 +95,15 @@ ezpublish_rest_copyContent:
     requirements:
         contentId: \d+
 
+ezpublish_rest_deleteContentTranslation:
+    path: /content/objects/{contentId}/{languageCode}
+    defaults:
+        _controller: ezpublish_rest.controller.content:deleteContentTranslation
+    methods: [DELETE]
+    requirements:
+        contentId: \d+
+        languageCode: '[a-z]{3}-[A-Z]{2}'
```

Comment:
This path (`/content/objects/{contentId}/{languageCode}`) overlaps a little bit with other paths, so I've added this requirement to make sure route is matched for language code only.

As stated in the Jira issue, IMHO, it would be better to use here more consistent path, e.g. `/content/objects/{contentId}/translations/{languageCode}` to avoid conflicts in the future. 

Current path has been set due to [REST API v2 specification](https://github.com/ezsystems/ezpublish-kernel/blob/v6.10.1/doc/specifications/rest/REST-API-V2.rst#overview)